packages/core/package.json

```
@@ -29,7 +29,7 @@
     "!*.tsbuildinfo"
   ],
   "scripts": {
-    "lib:ses": "cp ../../node_modules/ses/dist/lockdown.umd.js ./lib/lockdown.umd.js",
+    "lib:ses": "cp ../../node_modules/ses/dist/lockdown.umd.js ./lib/lockdown.umd.js && git add -A ./lib/lockdown.umd.js",
```

```suggestion
    "lib:ses": "cp ../../node_modules/ses/dist/lockdown.umd.js ./lib/lockdown.umd.js",
```
I think the package scripts like this shouldn't be modifying the local git environment, any drawback to moving it into the lint-script?

---

sure (I had it this way originally but changed it for reasons that escape me)